src/tex/ms.tex

```diff
@@ -91,7 +91,9 @@ \section{Introduction}
 \end{equation}
 %
 The input for this refinement process is the model and some initial parameter values, which may be an absolute value or some parameter range, depending on the refinement algorithm. 
+% ARJN the output of a Bayesian modelling process is a probability distribution for x. If everything is nice these will be Normally distributed, but not necessarily. We then report some statistical measure, e.g. mode/mean/median.
```

Comment:
I worry that introducing both aspects (refinement and sampling) together is potentially confusing to the reader.